kvmagent/kvmagent/test/unittest_tools/unittest_pypi_source/pypi_source_builder.sh

```
@@ -0,0 +1,137 @@
+    local minio_path=$MINIO_URL
+    local requirement_file_path="$PACKAGE_LIST_PATH"  # requirements/requirements1.txt
+    local local_pipy_dir_path="$BUILD_TARGET_PATH"  # /tmp/pipy_requirements1
+
+    mkdir -p "$local_pipy_dir_path/simple"
+    local index_html_text=""
+
+    [ x"$PYPI_REQUIREMENT_OUTPUT" = x"true" ] && touch $PYPI_REQUIREMENT_OUTPUT_PATH
+    local pypi_requirement_text=""
+
+    while IFS= read -r line || [[ -n "$line" ]]; do
+        local package_name=$(echo "$line" | xargs) # ex: Cython-0.29.37.tar.gz
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)